app/components/ui/Modal/ServiceDiscoveryModal.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react';
+import { BaseModal } from './Modal';
+import * as dataService from '../../../utils/DataService';
+
+import styles from './ServiceDiscoveryModal.scss';
+
+
+class ServiceDiscoveryModal extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      eligibilities: [],
+      subcategories: [],
+      checkedItems: {},
+    };
+  }
+
+  componentDidMount() {
+    const { categoryId } = this.props;
+
+    dataService.get(`api/eligibilities?category_id=${categoryId}`).then(response => {
+      const { eligibilities } = response;
+      this.setState({
+        eligibilities,
+      });
+    });
+
+    dataService.get(`api/categories/subcategories?id=${categoryId}`).then(response => {
+      const { categories } = response;
+      this.setState({
+        subcategories: categories,
+      });
+    });
+  }
+
+  handleCheckboxClick(optionId) {
+    const { checkedItems } = this.state;
+    checkedItems[optionId] = !checkedItems[optionId];
+  }
+
+  render() {
+    const { isEligibility } = this.props;
```

Comment:
We'll pass this prop to determine whether we show the eligibility options or the subcategory options, but I still have an open question about how we determine which to use here: https://sheltertech.slack.com/archives/C013NS0057A/p1590026695002300